src/components/PageWrapper/page-wrapper.css

```diff
@@ -0,0 +1,32 @@
+.green-bg {
+  background: linear-gradient(
+    128.9deg,
+    #76dd78 -18.28%,
+    rgba(118, 221, 120, 0.15) 63.69%
+  );
+  height: 100vh;
+}
+
+.wrapper {
+  background: #ffffff;
+  border: 1px solid #ffffff;
+  box-sizing: border-box;
+  box-shadow: 0px 4px 24px rgba(0, 0, 0, 0.17);
+  border-radius: 50px;
+  max-width: 600px;
+  width: 50%;
+  height: 80vh;
```

Comment:
Similarly, you may want to nerf out this height property completely, as it's allowing list items to overflow the wrapper:
<img width="947" alt="pagewrapperbug" src="https://user-images.githubusercontent.com/82783679/155808337-a0e65c31-4c4e-4c41-8106-554f7da20e49.png">

Without it, the wrapper will expand to fit the content, unless we're placing the list items in a scrollable div so that the webpage itself isn't vertically scrollable.

Reply:
Ahhh I totally missed this, thanks for catching it!!